official/resnet/resnet_run_loop.py

```diff
@@ -266,14 +266,69 @@ def learning_rate_fn(global_step):
                      false_fn=lambda: lr)
     return lr
 
+
+
+  def poly_rate_fn(global_step):
+    """Handles linear scaling rule, gradual warmup, and LR decay.
+
+    The learning rate starts at 0, then it increases linearly per step.  After
+    FLAGS.poly_warmup_epochs, we reach the base learning rate (scaled to account
+    for batch size). The learning rate is then decayed using a polynomial rate
+    decay schedule with power 2.0.
+
+    Args:
+    current_epoch: `Tensor` for current epoch.
+    poly_rate: Polynomial decay rate.
+
+    Returns:
+
+    A scaled `Tensor` for current learning rate.
+    """
+
+    # Learning rate schedule for LARS polynomial schedule
+    if flags.FLAGS.batch_size == 16384:
```

Comment:
Can you make these include the intermediate values? So for instance:
```
if flags.FLAGS.batch_size < 8192:
  plr = 5.0
  w_epochs = 5
elif flags.FLAGS.batch_size < 16384:
  plr = 10.0
  w_epochs = 5
...
```

Reply:
done